run/run.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/python3
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import configparser
+import gzip
+import json
+import platform as host_platform
+import re
+import requests
+import subprocess
+import sys
+import os
```

Comment:
I ran pep8 on this PR and I will fix any lint errors that come up. I made a bug #67 for introducing pep8 into our CI.